tests/test_input.py

```diff
@@ -0,0 +1,11 @@
+from dateparser.__init__ import parse
+from tests import BaseTestCase
+
+class InputTest(BaseTestCase):
+    def empty_input_test(self):
+        result = parse('')
+        self.assertFalse(result)
+
+    def whitespace_input_test(self):
+        result = parse(' \t\n\r\f\v')
+        self.assertFalse(result)
```

Comment:
Could we have a test case to check out **Null Bytes**?

Reply:
Do you mean add a special condition? Something like:
`if (date_string == '\x00'): return`

Reply:
Yes :)

Reply:
Got it! ;)